source/val/validate_bitwise.cpp

```diff
@@ -30,14 +30,14 @@ spv_result_t ValidateBaseType(ValidationState_t& _, const Instruction* inst,
 
   if (!_.IsIntScalarType(base_type) && !_.IsIntVectorType(base_type)) {
     return _.diag(SPV_ERROR_INVALID_DATA, inst)
-           << _.VkErrorID(4781)
```

Comment:
@alan-baker the one test, I removed the VUID because of the fact I removed it here

This is not a Vulkan VU, the Base being a int scalar/vector is part of the SPIR-V spec